### PR TITLE
fix: streaming disconnect after delay

### PR DESCRIPTION
**Pull request by nplasterer**
After a lot of reading and trial error I found that the tests will pass if we remove this line.
I even bumped the test up to a 300 second delay to make sure it still passes and confirmed that it does.

### CODE REVIEW

File: xmtp_api_grpc/src/lib.rs [hunk 1]
```diff
@@ -278,7 +278,7 @@ mod tests {
                 panic!("Message 1 Error: {}", err);
             }
 
-            tokio::time::sleep(std::time::Duration::from_secs(15)).await;
+            tokio::time::sleep(std::time::Duration::from_secs(50)).await;
```
Comment (nplasterer) [inline xmtp_api_grpc/src/lib.rs:281]: I bumped this up to 300 and it still passed with this change.

File: xmtp_api_grpc/src/grpc_api_helper.rs [hunk 2]
```diff
@@ -33,7 +33,6 @@ async fn create_tls_channel(address: String) -> Result<Channel, Error> {
         .map_err(|e| Error::new(ErrorKind::SetupError).with(e))?
         .keep_alive_while_idle(true)
         .connect_timeout(Duration::from_secs(5))
-        .http2_keep_alive_interval(Duration::from_secs(10))
```
Comment (nplasterer) [inline xmtp_api_grpc/src/grpc_api_helper.rs:36]: I bumped this all the way down to 1 and it still wouldn't pass. Removing it seems to be the only way to make it pass. Do we need this?

Comment (neekolas) [reply xmtp_api_grpc/src/grpc_api_helper.rs:36]: I thought the default of 20 seconds would kick in if we removed this setting, which might break this some time later than 50 seconds. 

Maybe I'm wrong about the defaults

Comment (nplasterer) [reply xmtp_api_grpc/src/grpc_api_helper.rs:36]: I set it to 300 seconds and it still didn't break. I can try even longer.

Comment (nplasterer) [reply xmtp_api_grpc/src/grpc_api_helper.rs:36]: Okay just set the delay to 1000 seconds which is like 16 minutes. And it still passed. So I think this is safe.

**Review by neekolas — APPROVED** [review]

**Review by alexrisch — APPROVED** [review]

File: xmtp_api_grpc/src/grpc_api_helper.rs (hunk 2 above)
Comment (neekolas) [reply xmtp_api_grpc/src/grpc_api_helper.rs:36]: There's a risk that this is going to break the disconnect detection on mobile, but that feels less bad than not streaming at all.

**Pull request merged**